test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -404,15 +421,18 @@ export class DashboardPageControls extends FtrService {
     await this.testSubjects.click(`field-picker-select-${fieldName}`);
   }
 
-  public async controlEditorSetType(type: string) {
-    this.log.debug(`Setting control type to ${type}`);
-    await this.testSubjects.click(`create-${type}-control`);
+  public async controlEditorVerifyType(type: string) {
+    this.log.debug(`Verifying that control type has type ${type}`);
+    const autoSelectedType = await this.testSubjects.getVisibleText('control-editor-type');
+    expect(autoSelectedType).to.equal(CONTROL_DISPLAY_NAMES[type]);
```

Comment:
By adding this `expect` here, I am always verifying the type whenever a new control is created and/or edited - so, don't really need to add any specific tests for field-first creation since all existing tests are now consistently checking that the expected types are being chosen.

Reply:
This is a great way to implicitly test this change. Really nice!